lib/src/descriptor.pb.dart

```diff
@@ -180,7 +182,7 @@ class DescriptorProto_ExtensionRange extends GeneratedMessage {
   static DescriptorProto_ExtensionRange _defaultInstance;
   static void $checkItem(DescriptorProto_ExtensionRange v) {
     if (v is! DescriptorProto_ExtensionRange)
-      checkItemFailed(v, 'DescriptorProto_ExtensionRange');
+      checkItemFailed(v, _i.messageName);
```

Comment:
should this messageName be
        google.protobuf.ExtensionRange
 or 
        google.protobuf.DescriptorProto.ExtensionRange
?